force-app/main/default/lwc/rd2ElevateCreditCardForm/rd2ElevateCreditCardForm.js

```diff
@@ -82,6 +102,13 @@ export default class rd2ElevateCreditCardForm extends LightningElement {
     @api payerLastName;
     @api achAccountType;
 
+    @wire(getObjectInfo, { objectApiName: ACCOUNT_OBJECT })
```

Comment:
Is there a better way (or a better place) to get this field label?

Reply:
I think this is ok since we're not driving any critical functionality from this wire, the only thing that we're grabbing is a label to display. One nitpick here for me would be that we should import the BillingAddress field and use that import to grab the Billing Address field info out of the ObjectInfo. I realize this field will basically never change name or be namespaced, but I think it's a good habit to have whenever we're working with field names.